demo/index.md

```diff
@@ -67,7 +67,81 @@ A baseline `<auro-select>` using `<auro-menu>` and `<auro-menuoption>` elements.
 <!-- AURO-GENERATED-CONTENT:END -->
 </auro-accordion>
 
-### Example with auro-icons in options
+## Nested example
```

Comment:
Let's update this to have three buttons opening separate dialogs. One that opens a version with the overflow: visible and one without. The third should have enough content in it to cause the UI issue that is the potential problem the user needs to watch out for.